cupy/cuda/function.pyx

```diff
@@ -122,6 +43,8 @@ cdef _launch(size_t func, Py_ssize_t grid0, int grid1, int grid2,
     cdef CPointer cp
     kargs.reserve(len(args))
     for a in args:
+        if a is None:
+            kargs.push_back(<void*>0)
```

Comment:
Is this relevant to this PR?

Reply:
Yes.

Reply:
Could you explain how?

Could you also answer this question?
>Is the title "Reduce Python object to improve kernel launch performance" still relevant?
>Could you explain where Python objects are reduced, or fix the title accordingly?

Reply:
OK, I changed PR title.